aws-crt/Rakefile

```diff
@@ -12,15 +13,13 @@ task :compile do
   build_dir = File.expand_path('build', native_dir)
   FileUtils.mkdir_p(build_dir)
   Dir.chdir(build_dir) do
-    if !File.exist?('CMakeCache.txt')
-      sh "cmake #{native_dir}"
-    end
+    sh "cmake #{native_dir}" unless File.exist?('CMakeCache.txt')
     sh "cmake --build #{build_dir}"
   end
 end
 
 desc 'Move the compiled lib into bin'
-task :bin => :compile do
+task bin: :compile do
```

Comment:
trivial: I have to say, I liked it better with the arrows
can we ignore that rule in this file?

Reply:
This style has been proposed a few times, eg: https://github.com/rubocop-hq/rubocop/issues/1437 and https://github.com/rubocop-hq/rubocop/issues/519 but hasn't really gained support.  

I agree I prefer the `=>` (aka hash rocket) for defining tasks as I find it more readable.  I'll disable this check for just the Rakefile.